src/components/accordion-item/accordion-item.tsx

```diff
@@ -101,17 +102,24 @@ export class AccordionItem {
 
     const iconEl = <calcite-icon class="accordion-item-icon" icon={this.icon} scale="s" />;
 
+    const { guid } = this;
+    const regionId = `${guid}-region`;
+    const buttonId = `${guid}-button`;
+
     return (
-      <Host aria-expanded={this.active.toString()} tabindex="0">
+      <Host tabindex="0">
         <div
           class={{
             [`icon-position--${this.iconPosition}`]: true,
             [`icon-type--${this.iconType}`]: true
           }}
         >
           <div
+            aria-controls={regionId}
             class={{ "accordion-item-header": true, [CSS_UTILITY.rtl]: dir === "rtl" }}
+            id={buttonId}
             onClick={this.itemHeaderClickHandler}
+            role="button"
```

Comment:
Maybe we should just make this a button element in the future? We should make sure there are no nested button element or roles as well.

Reply:
https://github.com/Esri/calcite-components/issues/4123